app/cmd/root.go

```diff
@@ -27,18 +27,14 @@ var rootCmd = &cobra.Command{
 				  Manage your Jenkins and your pipelines
 				  More information could found at https://jenkins-zh.cn`,
 	Run: func(_ *cobra.Command, _ []string) {
-		fmt.Println("Jenkins CLI (jcli) manage your Jenkins")
 
-		current := getCurrentJenkinsFromOptionsOrDie()
-		if current != nil {
-			fmt.Println("Current Jenkins is:", current.Name)
-		} else {
-			fmt.Println("Cannot found the configuration")
-		}
-
-		if rootOptions.Version {
-			fmt.Printf("Version: %s\n", app.GetVersion())
-			fmt.Printf("Commit: %s\n", app.GetCommit())
+		if !rootOptions.Version {
+			current := getCurrentJenkinsFromOptionsOrDie()
+			if current != nil {
+				fmt.Println("Current Jenkins is:", current.Name)
```

Comment:
Consider using `cmd.Println()` instead of `fmt.Println()`. It's convenice for adding test cases.
See also https://github.com/jenkins-zh/jenkins-cli/blob/master/app/cmd/job_search.go#L57